spec.bs

```diff
@@ -2827,6 +2841,92 @@ Note: {{ScoringBrowserSignals}}'s {{ScoringBrowserSignals/adComponents}} is {{un
 [=generated bid/ad component descriptors=] is null or [=list/is empty|an empty list=]. It cannot be
 an [=list/is empty|empty list=].
 
+<xmp class="idl">
+
+dictionary ReportingBrowserSignals {
+  required DOMString topWindowHostname;
+  required USVString interestGroupOwner;
+  required USVString renderURL;
+  required double bid;
+  required DOMString bidCurrency;
+  required double highestScoringOtherBid;
+  required DOMString highestScoringOtherBidCurrency;
+
+  unsigned long dataVersion;
+  USVString topLevelSeller;
+  USVString componentSeller;
+};
+</xmp>
+
+{{ReportingBrowserSignals}} includes browser signals both `reportResult()` and `reportWin()` get.
+The meanings and notes of some fields:
+<dl class=domintro>
+  <dt>{{ReportingBrowserSignals/renderURL}}
+  <dd>The render URL returned by "`generateBid()`". It is
+    [=query reporting ID k-anonymity count|k-anonymous=]
+  <dt>{{ReportingBrowserSignals/bid}}
+  <dd>The [=round a value|rounded value=] of the winning bid. This is always in the bidder's own
+    currency.
+  <dt>{{ReportingBrowserSignals/highestScoringOtherBid}}
+  <dd>The [=round a value|rounded value=] of the the bid that got the second highest score, or 0 if
+    it's not available This is always in the bidder's own currency.
+</dl>
+
+<xmp class="idl">
+
+dictionary ReportResultBrowserSignals : ReportingBrowserSignals {
+  required double desirability;
+
+  DOMString topLevelSellerSignals;
+  double modifiedBid;
+};
+</xmp>
+
+The meanings and notes of some fields:
+<dl class=domintro>
+  <dt>{{ReportResultBrowserSignals/desirability}}
+  <dd>The [=round a value|rounded value=] of the score returned by "`scoreAd()`" for the winning bid.
+  <dt>{{ReportResultBrowserSignals/topLevelSellerSignals}}
+  <dd>Metadata returned by the top-level seller's "`reportResult()`", as JSON.
+  <dt>{{ReportResultBrowserSignals/modifiedBid}}
+  <dd>The [=round a value|rounded value=] of the bid value returned by the component seller's
+    "`scoreAd()`" method
+  <dt>{{ReportingBrowserSignals/dataVersion}}
+  <dd>Only set if the Data-Version header was provided in the response headers from the trusted
+    scoring signals server.
+</dl>
+
+<xmp class="idl">
+
+dictionary ReportWinBrowserSignals : ReportingBrowserSignals {
+  double adCost;
+  USVString seller;
+  boolean madeHighestScoringOtherBid;
+  DOMString interestGroupName;
+  unsigned short modelingSignals;
+};
+</xmp>
+
+The meanings and notes of some fields:
+<dl class=domintro>
+  <dt>{{ReportWinBrowserSignals/adCost}}
+  <dd>Advertiser click or conversion cost passed from `generateBid()` to `reportWin()`. Non-negative
+    and only the lowest 8 bits will be passed

```

Comment:
I'm not sure what the "lowest 8 bits" means for a double. Is this the rounding operation from above?